seed/challenges/03-front-end-libraries/jquery.json

```diff
@@ -145,7 +145,7 @@
       "tests": [
         "assert($(\"button\").hasClass(\"animated\") && $(\"button\").hasClass(\"bounce\"), 'message: Use the jQuery <code>addClass&#40&#41</code> function to give the classes <code>animated</code> and <code>bounce</code> to your <code>button</code> elements.');",
         "assert(!code.match(/class.*animated/g), 'message: Only use jQuery to add these colors to the element.');",
-        "assert(/'/g.test(code) === false , 'message: Only use double quotes in your jQuery code, as is stated in the jQuery style guide.');"
+        "assert(/'/g.test(code) === false , 'message: Try using double quotes in your jQuery code, as described in challenge description');"
```

Comment:
You haven't changed the challenge test, so this still forces people to use double quotes. Just a note here. I'll post in the original issue to continue the discussion.